decidim-budgets/app/models/decidim/budgets/order.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module Decidim
+  module Budgets
+    # The data store for a Order in the Decidim::Budgets component. It is unique for each
+    # user and feature and contains a collection of projects
+    class Order < Budgets::ApplicationRecord
+      belongs_to :user, class_name: Decidim::User, foreign_key: "decidim_user_id"
+      belongs_to :feature, class_name: Decidim::Feature, foreign_key: "decidim_feature_id"
```

Comment:
+1 to use `HasFeature` but I prefer not using `Authorable` because we don't need it here.